packages/office-ui-fabric-react/src/components/Callout/Callout.types.ts

```diff
@@ -227,6 +227,14 @@ export interface ICalloutProps extends React.HTMLAttributes<HTMLDivElement> {
    * once the callout is visible.
    */
   hidden?: boolean;
+
+  /**
+   * If specified, determines whether the underlying "Popup" component should try to restore
+   * focus when it is dismissed.  When set to false, the Popup won't try to restore focus to
+   * the last focused element.
+   * Defaults to true when undefined.
```

Comment:
To ensure that the default value shows up the the API references table, could you add a `@defaultvalue true` line to the TSDoc comment? Similar to what we have on `beakWidth`, for example:

https://github.com/OfficeDev/office-ui-fabric-react/blob/843301fa058f5b7c89c753c90baa2812edf1eed2/packages/office-ui-fabric-react/src/components/Callout/Callout.types.ts#L42-L46

Reply:
Sounds good!